AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.spec.ts

```diff
@@ -664,117 +662,6 @@ describe('BookingsListComponent', () => {
         expect(component.loaded).toBeTruthy();
     });
 
-    it('should onSearch (admin_search flag off)', () => {
-        bookingsListServiceSpy.getBookingsList.calls.reset();
-        setFormValue();
-        component.enableSearchFeature = false;
-        component.onSearch();
-        expect(bookingPersistService.caseNumber).toMatch('CASE_NUMBER');
-        expect(bookingPersistService.startDate).toEqual(moment().startOf('day').add(1, 'days').toDate());
-        expect(bookingPersistService.endDate).toEqual(moment().startOf('day').add(2, 'days').toDate());
-        expect(component.bookings.length).toBeGreaterThan(0);
-        expect(bookingsListServiceSpy.getBookingsList).toHaveBeenCalledWith(undefined, component.limit);
-    });
-
-    it('should onSearch (admin_search flag on)', () => {
```

Comment:
please can we keep these tests, but remove the `component.enableSearchFeature = true`

can also take the "admin_search flag on" out of the titles

Reply:
done